src/Symfony/Component/Translation/Dumper/XliffFileDumper.php

```diff
@@ -40,6 +50,21 @@ public function dump(MessageCatalogue $messages, $options = array())
     }
 
     /**
+     * Set XLIFF tool element metadata.
+     *
+     * @link http://docs.oasis-open.org/xliff/v1.2/os/xliff-core.html#tool_elem
+     *
+     * @param array attributes supported by the <tool> element
+     */
+    protected function setToolMetadata( array $meta ){
```

Comment:
I tend to agree with @stof, instead of introducing a new method here we can override those value via [$options](https://github.com/timwhitlock/symfony/blob/ticket_15190/src/Symfony/Component/Translation/Dumper/XliffFileDumper.php#L41)